trace-dispatcher/examples/Examples/Aggregation.hs

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE DeriveGeneric       #-}
+{-# LANGUAGE FlexibleInstances   #-}
+{-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving  #-}
+
+
+module Examples.Aggregation (
+  testAggregation
+) where
+
+import qualified Data.Aeson as A
+import           GHC.Generics (Generic)
+
+import           Cardano.Logging
+
+data BaseStats = BaseStats {
+    bsMeasure :: Double,
+    bsMin     :: Double,
+    bsMax     :: Double,
+    bsCount   :: Int,
+    bsSum     :: Double
+    } deriving (Eq, Ord, Show, Generic)
+
+instance A.ToJSON BaseStats where
+    toEncoding = A.genericToEncoding A.defaultOptions
+
+instance LogFormatting BaseStats where
+  forMachine = mempty
+  asMetrics BaseStats {..} =
+    [ DoubleM ["measure"] bsMeasure
+    , DoubleM ["sum"] bsSum]
+
+baseStatsDocumented :: Documented Double
+baseStatsDocumented =
+  Documented [
+    DocMsg 0.0 [] "This is the value of the measurement"
+  , DocMsg 0.0 [] "This is the sum of all measurments so far"
+  ]
```

Comment:
Please choose some style for `[` and `]` placing and use it everywhere. I mean the position of `[`and `]`:

```
instance LogFormatting BaseStats where
  forMachine = mempty
  asMetrics BaseStats {..} =
     [ DoubleM ["measure"] bsMeasure
     , DoubleM ["sum"] bsSum
     ]

baseStatsDocumented :: Documented Double
baseStatsDocumented =
  Documented
    [ DocMsg 0.0 [] "This is the value of the measurement"
    , DocMsg 0.0 [] "This is the sum of all measurments so far"
    ]
```

Reply:
Resolved by commit `Unifying mutiline list layout`